worker/create_batches_from_filters_worker.go

```diff
@@ -268,39 +291,65 @@ func (b *CreateBatchesFromFiltersWorker) Process(message *workers.Msg) {
 		zap.String("jobID", id.String()),
 	)
 	l.Info("starting create_batches_using_filters_worker")
+
 	job := &model.Job{
 		ID: id,
 	}
+	job.TagRunning(b.MarathonDB, "create_batches_using_filters_worker", "starting")
 	err = b.MarathonDB.DB.Model(job).Column("job.*", "App").Where("job.id = ?", job.ID).Select()
-	checkErr(l, err)
+	if err != nil {
+		job.TagError(b.MarathonDB, "create_batches_using_filters_worker", err.Error())
```

Comment:
`create_batches_using_filters_worker` is repeated several times, maybe create a variable?

Reply:
I create one checkErr method for the CreateBatchesFromFiltersWorker struct to don't repeat the error sentence.

Reply:
We still have `create_batches_using_filters_worker` in a few places. Even with the checkErr method (nice catch by the way) I'd have a variable with this string.